client_cli/src/main.rs

```diff
@@ -683,13 +704,19 @@ mod asset {
         /// List assets
         #[clap(subcommand)]
         List(List),
+        /// Put a key-value in a store Asset defintion
+        SetKeyValue(SetKeyValue),
+        /// Remove a key-value via key in a store Asset definition
+        RemoveKeyValue(RemoveKeyValue),
+        /// Get a value from store Asset definition via key
+        GetKeyValue(GetKeyValue),
```

Comment:
I think there commands work with actual assets, not their definitions.

We should ask tech writers (@outoftardis, @yamkovoy) how to document these commands better.

Maybe,

> Create a key-value entry in a store asset

or

> Remove Store asset entry

?

Reply:
using "Store asset" instead of "store asset definition" should be fine